### DISCUSSION TITLE
Events that repeat weekly are handled differently than Events that repeat on select dates

### DISCUSSION THREAD

**Discussion by laurabennett** [solspace/craft-calendar#283] [Questions & Answers]
I would like to display all instances of an event. My template follows the documentation.
https://docs.solspace.com/craft/calendar/v4/templates/objects/occurrences/

For weekly event the first occurrence to display includes the start date.
For events that only recur on select dates the start date is not included in the query.  

specifically the occurenceCache for an event where freq:"WEEKLY" includes an event with the start date and the occurenceCache for an event where freq:"SELECT_DATES" does not include an event with the start date, the first event is the repeats on select date.

**Steps to reproduce**

1. Create two events
2. weekly recurring event (all day)
3. event with start and two additional dates (all day) ( note the start date is not available to select when choosing select dates)
4. twig template follows documentation for displaying upcoming recurrences

**Expected behavior**
Upcoming occurrences includes the start date.
If I need to query the start date separately then the weekly event should also not include the start date.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: Pro 4.5.14
- Calendar Version: 4,0.11
- Calendar Edition: Pro

**Additional context**

**Comment by kjmartens** [comment]
Sorry for the long delay @laurabennett,

This is expected behavior. The trouble is that many customers expected that the Start Date would not be included... and only the "Select Dates" would be. It's a bit of a tricky one, but I have noted this and we'll see if we can improve this. 🙂
